src/commands/outdated/OutdatedCommand.php

```diff
@@ -73,7 +73,7 @@ private function renderCliOutput(array $outdated): string {
 
         return \sprintf(
             "Found %d outdated PHARs in phive.xml:\n\n%s",
-            $outdated,
+            count($outdated),
```

Comment:
@theseer This is critical.

Reply:
Already fixed locally as psalm complained as well ;)